OpenEdXMobile/src/main/java/org/edx/mobile/model/db/DownloadEntry.java

```diff
@@ -48,6 +48,9 @@
     public String lmsUrl;
     public TranscriptModel transcript;
 
+    public String castUrl;
+    public String videoThumbnail;
```

Comment:
> And we should rename it to courseThumbnail

I think **`videoThumbnail`** is more appropriate because the thumbnail is casting with the video and in the future, we don't need to rename it when video thumbnail is available.